core/testcasecontroller/simulation_system_admin/simulation_system_admin.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2022 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+
+def check_host_docker():
+    # check whether Docker is installed on the host.
+    # If Docker is not installed, try to install Docker with one-click installation script.
+    shell_cmd = "docker version | head -n 2"
+    check_docker = subprocess.run(shell_cmd, shell=True)
+
+    if check_docker.returncode != 0:
+        # trying to install docker
+        print("trying to install docker")
+        try:
+            shell_install_docker_cmd = "curl -fsSL https://get.docker.com | bash -s docker --mirror Aliyun"
+            install_docker = subprocess.run(shell_install_docker_cmd, shell=True)
+
+            if install_docker.returncode == 0:
+                print("successfully installed docker")
+            else:
+                raise Exception(f"install docker failed")
+        except Exception as err:
+            raise Exception(f"install docker failed, error: {err}.") from err
+
+    print("check docker successful")
+
+def check_host_kind():
+    # check whether Docker is installed on the host.
+    shell_cmd = "kind version"
+    check_kind = subprocess.run(shell_cmd, shell=True)
+
+    if check_kind.returncode != 0:
+        raise Exception(f"Kind is not installed. You need to install Kind")
```

Comment:
Why not help users install kind?

Reply:
Heard that only need one command 👯‍♂️

Reply:
Thanks. I've added an automatic kind installation script to the latest code.

Reply:
But I have a concern, is that the automatic installation of kind may require root permissions. Because the shell will write binary package to the `/usr/local/bin` path.